continuationsPlugin/src/main/scala/continuations/Continuation.scala

```diff
@@ -13,6 +13,12 @@ object Continuation:
   enum State:
     case Suspended, Undecided, Resumed
 
+  /**
+   * No body needed as the compiler plugin replaces this in call sites
+   */
+  inline def suspendContinuation[A](f: Continuation[A] => Unit)(using s: Suspend): A =
+    ???
```

Comment:
should probably throw a proper exception unless the plugin synthetically generates this function. If it lives in the classpath compiled and we replace just calls to it, a user may link against the runtime lib without enabling the compiler plugin.

Reply:
Yeah, I think this should probably be a `Synthetic`. The question is... where do we initialize it? Probably in a prep miniphase `prepareForUnit` -- we only need to do it one time. Ditto for anything we have marked as `???` that we are going to generate. We only need it for typechecking, yeah?

Reply:
New card created to address this: https://app.clickup.com/t/371zuqx